src/Command/ImportCommand.php

```diff
@@ -10,28 +11,33 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
 
-class ImportCommand extends Command {
+class ImportCommand extends Command
+{
     protected static $defaultName = 'import:repository';
 
-    private $githubService;
     private $repoRepository;
-    private $bitbucketService;
     private $providers;
-    
-    public function __construct(GithubService $githubService, RepoRepository $repoRepository, bitbucketService $bitbucketService, ParameterBagInterface $params)
+    private $importCommandCreator;
+    private $bus;
+
+    public function __construct(MessageBusInterface $bus, ImportCommandCreator $importCommandCreator, RepoRepository $repoRepository, ParameterBagInterface $params)
```

Comment:
I don't see $repoRepository usage so we can remove it from  constructor.
The same with params. We only use it to check if given provider is handled by us. Instead we could delegate it to our ImportCommandCreator. If non command was created let throw exception. I will comment it on the ImportCommandCreator

Reply:
Fixed